<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отбор резких изображений</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/icons/favicon.ico</normaloff>:/icons/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="view_tab">
       <attribute name="title">
        <string>Просмотр</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGraphicsView" name="vie_gv">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="diff_tab">
       <attribute name="title">
        <string>Вычетание</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGraphicsView" name="diff_gv"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_openDir"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_previous"/>
    <addaction name="action_next"/>
    <addaction name="action_first"/>
    <addaction name="action_last"/>
    <addaction name="separator"/>
    <addaction name="action_zoomIn"/>
    <addaction name="action_zoomOut"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="settings_dock">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>559</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Настройки</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="settings_dockCtnt">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="dir_gb">
       <property name="title">
        <string>Выбор папки</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="dirPath_H_lyt">
          <item>
           <widget class="QLineEdit" name="dirPath_le"/>
          </item>
          <item>
           <widget class="QToolButton" name="dirPath_btn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="dir_H_line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="imgsSizeCnt_lyt">
          <item row="0" column="0">
           <widget class="QLabel" name="imgSize_lbl">
            <property name="text">
             <string>Размер</string>
            </property>
            <property name="buddy">
             <cstring>imgSizeW_sb</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="imgSize_H_lyt">
            <item>
             <widget class="QSpinBox" name="imgSizeW_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string> px</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="imgSizeX_lbl">
              <property name="text">
               <string>x</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>imgSizeH_sb</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="imgSizeH_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string> px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="imgCnt_lbl">
            <property name="text">
             <string>Количество</string>
            </property>
            <property name="buddy">
             <cstring>imgCnt_sb</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="imgCnt_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> шт</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="dir_progress">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="view_gb">
       <property name="title">
        <string>Отображение</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="tl_H_lyt">
          <item>
           <widget class="QLabel" name="tl_lbl">
            <property name="text">
             <string>Лево верх</string>
            </property>
            <property name="buddy">
             <cstring>tlx_sb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tlx_sb">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>( </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tlx_lbl">
            <property name="text">
             <string>;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buddy">
             <cstring>tly_sb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tly_sb">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> )</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="br_H_lyt">
          <item>
           <widget class="QLabel" name="tl_lbl_2">
            <property name="text">
             <string>Право низ</string>
            </property>
            <property name="buddy">
             <cstring>brx_sb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="brx_sb">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>( </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tlx_lbl_2">
            <property name="text">
             <string>;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buddy">
             <cstring>bry_sb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="bry_sb">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> )</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="scale_H_lyt">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Масштаб</string>
            </property>
            <property name="buddy">
             <cstring>scale_sldr</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="scale_sldr">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scale_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="diff_gb">
       <property name="title">
        <string>Вычетание</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="baseImg_H_lyt">
          <item>
           <widget class="QLabel" name="baseImg_lbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Опорный кадр</string>
            </property>
            <property name="buddy">
             <cstring>baseImg_cb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baseImg_cb"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="baseImagPrevie_lbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="diffK_H_lyt">
          <item>
           <widget class="QLabel" name="diffK_lbl">
            <property name="text">
             <string>k = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>diffK_H_sldr</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="diffK_H_sldr">
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="diffK_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="settings_spcr">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>466</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_openDir">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/open_folder_ic.png</normaloff>:/icons/open_folder_ic.png</iconset>
   </property>
   <property name="text">
    <string>Открыть папку</string>
   </property>
   <property name="toolTip">
    <string>Открыть папку с исходными изображениямий</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/app_quit_ic.png</normaloff>:/icons/app_quit_ic.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход из приложения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_next">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/next_ic.png</normaloff>:/icons/next_ic.png</iconset>
   </property>
   <property name="text">
    <string>Следующее изображение</string>
   </property>
   <property name="toolTip">
    <string>Вывод на экран следующего по списку изображения</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="action_previous">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/previous_ic.png</normaloff>:/icons/previous_ic.png</iconset>
   </property>
   <property name="text">
    <string>Предыдущее изображение</string>
   </property>
   <property name="toolTip">
    <string>Вывод на экран предыдущего по списку изображения</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="action_first">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/first_ic.png</normaloff>:/icons/first_ic.png</iconset>
   </property>
   <property name="text">
    <string>Первое изображение</string>
   </property>
   <property name="toolTip">
    <string>Вывод на экран первого  по списку изображения</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="action_last">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/last_ic.png</normaloff>:/icons/last_ic.png</iconset>
   </property>
   <property name="text">
    <string>Последнее изображение</string>
   </property>
   <property name="toolTip">
    <string>Вывод на экран последнего по списку изображения</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="action_zoomIn">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/zoom_in_ic.png</normaloff>:/icons/zoom_in_ic.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="toolTip">
    <string>Увеличить масштаб изображения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_zoomOut">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/zoom_out_ic.png</normaloff>:/icons/zoom_out_ic.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить масштаб изображения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dirPath_btn</sender>
   <signal>clicked()</signal>
   <receiver>action_openDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diffK_H_sldr</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>diffK_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diffK_H_sldr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>diffK_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
